pkg/database/models.go

```diff
@@ -4,9 +4,18 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/gookit/validate"
+	"golang.org/x/crypto/bcrypt"
 	"gorm.io/gorm"
 )
 
+type RoleType string
+
+const (
+	RoleUser  RoleType = "user"
+	RoleAdmin RoleType = "admin"
```

Comment:
The if the type is the same, the second line doesnt need to have it
```suggestion
	RoleAdmin  = "admin"
```

Reply:
fixed